recipe/meta.yaml

```diff
@@ -29,10 +29,18 @@ outputs:
       host:
         - python
         - pip
+        - docutils
+        - requests
         - setuptools
+        - tomli
+        - tomli-w
         - wheel
       run:
         - python >=3.6
+        - docutils
+        - requests
+        - tomli
+        - tomli-w
```

Comment:
These are dependencies for `flit` not flit-core. Both the [upstream pyproject.toml](https://github.com/pypa/flit/blob/3.6.0/flit_core/pyproject.toml#L12) and [conda-forge recipe](https://github.com/conda-forge/flit-feedstock/blob/a61caf2c0f1f84e1c8ef20de517f3c4192538923/recipe/meta.yaml#L23-L37) indicate that flit-core's only dependencies should be python and pip.

Reply:
The flit dependencies have beenn removed with the exception of `tomli`

https://github.com/AnacondaRecipes/flit-feedstock/blob/d16c30ce386334db24089e5a7970ed67f92d1578/recipe/meta.yaml#L29&L39

The reason why we are rebuilding flit-core is because the current version that we have on defaults is missing `tomli`. If we plan on eliminating `tomli` and their related dependencies from `flit-core`, then we probably should use flit in the `pyparsing` package instead.